infogami/core/helpers.py

```diff
@@ -57,11 +57,11 @@ def setdefault(d, k, v):
         elif '#' in k:
             a, b = k.split('#', 1)
             return setdefault(setdefault(d, a, betterlist()), b, v)
-        else: 
+        else:
             return d.setdefault(k, v)
 
     d2 = {}
-    for k, v in d.iteritems():
+    for k, v in d.items():
```

Comment:
Used only once.  Python 2 will build a list in memory which will take up more RAM.  Python 3 will be fast.